libexec/rc/rc.conf

```diff
@@ -138,6 +138,7 @@ dhclient_flags=""		# Extra flags to pass to dhcp client.
 #dhclient_flags_em0=""		# Extra dhclient flags for em0 only
 background_dhclient="NO"	# Start dhcp client in the background.
 #background_dhclient_em0="YES"	# Start dhcp client on em0 in the background.
+dhclient_noarpwait="NO"	# Trust IP address from DHCP and don't ARP
```

Comment:
The double negative in `dhclient_noarpwait="NO"` reads awkwardly.  Reversing its logic, naming it `dhclient_wait_for_arp`, and setting its default value to `"YES"` might make it easier to understand.

Reply:
Good point.  I suggested "noarpwait" since that's what the `-n` flag means (and it needs to be that way around to keep the previous "no flags means wait for ARP" behaviour) but you're quite right that for a new rc.conf setting we can reverse that.

@ica574 How about `dhclient_arpwait="YES" # ARP IP addresses before using them` and a corresponding change in the rc.d script?

Reply:
Sounds good. I'll implement it!